locksmith/src/models/authorizedLock.ts

```diff
@@ -0,0 +1,11 @@
+import { Table, Column, Model } from 'sequelize-typescript'
+
+@Table({ tableName: 'AuthorizedLocks', timestamps: true })
+// eslint-disable-next-line import/prefer-default-export
+export class AuthorizedLock extends Model<AuthorizedLock> {
+  @Column
+  address!: string
```

Comment:
TS idiom for the uninitiated: the `!` is an assertion that a variable will not be null. The compiler can't always statically determine whether something will be defined or not, so this is us saying that we are 100% confident it will not be null.

https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#non-null-assertion-operator